src/io/flutter/logging/FlutterLogEntry.java

```diff
@@ -37,7 +37,10 @@
   private final int level;
   @NotNull
   private String message;
-  // TODO(pq): consider making data an Instance or JsonElement
+
+  /**
+   * Associated data; may be a JSON string value or diagnostic node.
```

Comment:
nit: DiagnosticsNode instead of diagnostic node to make clear what the actual union type is.

Reply:
👍 fixed.